jabgui/src/main/java/org/jabref/gui/LibraryTabContainer.java

```diff
@@ -27,7 +27,7 @@ public interface LibraryTabContainer {
      * Closes a designated libraryTab
      *
      * @param tab to be closed.
-     * @return true if closing the tab was successful
+     * @return true if closing the tab was isSuccessful
```

Comment:
IntelliJ probably replaced this
```suggestion
     * @return true if closing the tab was successful
```